src/main/java/com/minecolonies/coremod/entity/ai/basic/AbstractEntityAIBasic.java

```diff
@@ -1227,7 +1231,8 @@ private void updateToolFlag(@NotNull final String tool, final int required)
                 checkForPickaxe(required);
                 break;
             default:
-                Log.getLogger().error("Invalid tool " + tool + " not implemented as tool!");
+                checkForPickaxe(DIAMOND_LEVEL);
```

Comment:
The problem is that in many cases mods have special items to remove a block, like Railcraft etc.
Tools we often can't implement. Therefore, if there is a tool we didn't implement. Then -> use diamond pickaxe